docs/api.md

```diff
@@ -1380,6 +1384,155 @@ Exposes the Pino package version. Also available on the logger instance.
 
 * See [`logger.version`](#version)
 
+## Test
+Pino provides a test module with some useful APIs that you can use importing it as the following
+```js
+require('pino').test
+// OR
+const { test } = require('pino')
+```
+
+<a id="pino-test-sink"></a>
+### `pino.test.sink({ destroyOnError = false, emitErrorEvent = false }) => Transform`
+Create a stream using `split2` so that each line is a chunk and each chunk is JSON parsed.
```

Comment:
The purpose of this method should be explained. How the method does what it does is not useful information in the documentation.